maro/simulator/scenarios/vm_scheduling/business_engine.py

```diff
@@ -304,18 +355,53 @@ def _update_pm_workload(self):
                 vm = self._live_vms[vm_id]
                 total_pm_cpu_cores_used += vm.cpu_utilization * vm.cpu_cores_requirement
             pm.update_cpu_utilization(vm=None, cpu_utilization=total_pm_cpu_cores_used / pm.cpu_cores_capacity)
-            pm.energy_consumption = self._cpu_utilization_to_energy_consumption(cpu_utilization=pm.cpu_utilization)
+            if pm.cpu_utilization > 100:
```

Comment:
```
pm.energy_consumption = ...
if pm.cpu_utilization > 100:
    self._overload(pm_id=pm.id) # Move total_overload_pms += 1 to this function
```

How about this?
Then for this moment that overloading happens, the energy_consumption is still calculated with the utilization > 100

But if we kill all VMs in this PM, in the post_step() of this tick, the total_energy_consumption will just add the 0 or the part newly allocated VM counts?